### DISCUSSION TITLE
Invalid loop

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#1055] [Developers]
While installing new organ package i got this error (its fatal-red but organ was loaded and worked):
28/02/17:48:06: 17:47:37: Error: Invalid loop

pls what does it means ?

**Comment by oleg68** [comment]
It means that some wav file is invalid. I'll try to add it's filename to the message.

**Comment by larspalo** [comment]
That error message basically indicate that one of the loops in a wave file is invalid. It could be because the loop

1. start point is after end point
2. start point is after end of file
3. end point is after end of file
4. end point is sample 0

Basically the loop stored in the wave file is faulty in some way. If you get the file name that's causing the issue you can examine it and try to find what the issue is and possibly correct in with a suitable tool, like LoopAuditioneer or similar...

**Reply by paolo2504** [reply]
Thanks Lars. I’ll Try to fix this as soon as Oleg will be able to show the file name in the error msg (as he proposed).
Best regards.

**Reply by oleg68** [reply]
3.6.3-1 does it.

**Reply by paolo2504** [reply]
here we go... attached below the file. The sampleset is MelcerChamberMusicHall_GrandOrgue, downloadable from pgrabowsky website.
The error now show:
03/03/2022 12:59:34: 12:59:04: Error: Invalid loop in the file: Data - Melcer Chamber Music Hall\II Sifflote 1\Tremmed\R99999\050-d.wav

I opened the file with your loopauditioneer but i have first to study how it works...
[050-d.zip](https://github.com/GrandOrgue/grandorgue/files/8181895/050-d.zip)
